scripts/eosio_build.sh

```diff
@@ -108,6 +108,8 @@ if [ $# -ne 0 ]; then
    done
 fi
 
+[[ ! -d scripts ]] && echo "You must run this script from the eos root!" && exit 1
```

Comment:
Failing for arbitrary reasons is frustrating to users.
Instead of failing, we can automatically switch to the root of the repository for them as follows:
`[[ ! -d scripts ]] && cd $(git rev-parse --show-toplevel)`